MAINTAINERS.md

```diff
@@ -1,17 +1,91 @@
+
+#### NPM Account
+
+Sign up for an [NPM account](https://www.npmjs.com/signup) with
+some form of 2FA. Then request that an administrator add your NPM
+user ID to the `@agoric` organization and the `agoric` package.
+
+### Preparation
+
+Together with the Release Owner and other appropriate stakeholders,
+ensure there is consensus on the following parameters for the
+release:
+
+- Base branch: which branch will the release be based on? The base
+branch will need to be stable and may be frozen at various times,
+so it is a good practice to have the base branch be a branch off
+of `master`. The base branch might also be used for followon patch
+releases before the next major release.
+
+- Release labels: we currently label our releases with both a
+human-meaningful label such as `pismoA` and a sequential upgrade
+label like `agoric-upgrade-8`.  In the future, semantic versioning
+might replace either or both label styles.
+
+- Development history oriented release descrition: a short description
+of the major changes in the release from the point of view of
+understanding the source code and tracked issues addressed by the
+release.
+
+- Validator oriented release description: a short description of the
+release from the perspective of a validator to understand why the
+release should be installed and important operational changes.
+
+### Staging the Release
+
+Until we reach the next section "Publish the release", the release
+process can be aborted.
+
+- [ ] Check the development history oriented release description
+into the top-level CHANGELOG.md on the base branch.
+
+- [ ] Review the [next release](
 https://github.com/Agoric/agoric-sdk/labels/next%20release
 ) label for additional tasks particular to this release.
 
+- [ ] Create a fresh, clean clone of the repository.
+
+This avoids accidental contamination from untracked or ignored
+changes in one's usual development repository.
+
+```sh
+# Fresh checkout under ~/release or any other preferred directory without agoric-sdk
+cd ~/release
+git clone https://github.com/Agoric/agoric-sdk.git
+cd agoric-sdk
+```
+
+- [ ] Check out the appropriate base branch as outlined in "Preparation" above.
+
+- [ ] Create a release branch tagged with a timestamp. The specific commit tagged as the release will live
+in this branch. The release tags will be human-meaningful, the release branch need not be.
+
 ```sh
 # Create a release branch.
 now=`date -u +%Y%m%dT%H%M%S`
 git checkout -b release-$now
 ```
 
-To generate a new final release, and CHANGELOG.md files
-(use `--conventional-prerelease` instead of `--conventional-graduate` if you just want to generate a dev release):
+- [ ] Do a `yarn install` to generate tooling needed for the release.
+
+```sh
+# yarn install to build release tools
+yarn install --force
+```
+
+- [ ] Generate new SDK version and per-package CHANGELOG.md files
+
+Use `--conventional-prerelease` instead of `--conventional-graduate` if you just want to generate a dev release.
```

Comment:
good advice, but I don't know when/why we might want to generate a "dev release", or what exactly that means.. if you have a better idea, maybe we could add a few more words here.

Reply:
I don't have a better idea, so I invoke Chesterton's Fence to leave this alone.